resources/test/fixtures/B004.py

```diff
@@ -0,0 +1,12 @@
+def this_is_a_bug():
```

Comment:
Can we grab the test suite from flake8-bugbear?

Reply:
I took this code from:
https://github.com/PyCQA/flake8-bugbear/blob/main/tests/b004.py

Reply:
Oh, thanks!